lib/BloqadeQMC/src/BloqadeQMC.jl

```diff
@@ -18,6 +18,9 @@ using LinearAlgebra
 
 using Random
 
+import BloqadeLattices: rydberg_interaction_matrix, AtomList
+import BloqadeExpr: RydbergHamiltonian, get_rydberg_params, is_time_function
```

Comment:
same here, and ideally you should change ejaaz's old `import` statements accordingly, especially those from `Base`

Reply:
But if there is no time we can do it later

Reply:
Oh... I was messaging ejaaz yesterday and had concluded import is the proper way to do it :sweat_smile:

Reply:
no that's wrong I will convince Ejaaz next week about this. But `import` is definitely discouraged.